src/taskclass.js

```diff
@@ -0,0 +1,23 @@
+export default class Task {
+  constructor(taskEntry) {
+    this.taskEntry = taskEntry;
+    this.completed = false;
+  }
```

Comment:
- [ ] Well done implementing a task `completed` status. I think it would be better if a task has one more property which is the `index` and it sets to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). Below is a screenshot of your tasks array that might help you understand 👇 
![Screenshot (242)_LI](https://user-images.githubusercontent.com/67087664/161284523-4cd03498-4464-436e-b868-f7ea5dc86b95.jpg)